python/pyspark/__init__.py

```diff
@@ -112,7 +112,7 @@ def wrapper(self, *args, **kwargs):
 
 
 # for back compatibility
-from pyspark.sql import SQLContext, HiveContext, Row
+from pyspark.sql import SQLContext, HiveContext, Row  # noqa: F401
```

Comment:
@srowen I'd be happy to do some more cleanup, but ones like these can't be removed easily. They provide backward compatibility, and allows us to do the following:
```bash
fokkodriesprong@Fan spark % python3
Python 3.7.7 (v3.7.7:d7c567b08f, Mar 10 2020, 02:56:16) 
[Clang 6.0 (clang-600.0.57)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from pyspark import Row
>>> # This is allowed due to the import
```
For backward compatibility I would keep these, otherwise we might break a lot of imports.